src/breeze/actions/mixins/action_helpers.cr

```diff
@@ -3,15 +3,19 @@ module Breeze::ActionHelpers
     before store_breeze_request
     after store_breeze_response
 
-    Avram::Events::QueryEvent.subscribe do |event|
-      unless event.query.includes?("breeze_") || event.query.includes?("information_schema")
+    Avram::Events::QueryEvent.subscribe do |event, duration|
+      if Breeze.settings.enabled
```

Comment:
```suggestion
      next unless Breeze.settings.enabled
```

Could this just be a guard instead? I know some other might disagree but the rule of thumb that I keep in mind is that the "main" code shouldn't be nested if possible